src/main/java/seedu/duke/Duke.java

```diff
@@ -5,39 +5,41 @@
 import seedu.duke.ui.Ui;
 
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
```

Comment:
Is this package being imported because of the BusStops being set in the array?
Can this be removed?

Reply:
Yup, only for testing.

Reply:
```suggestion
```